starfish/image/_stack.py

```diff
@@ -446,7 +548,20 @@ def tile_opener(tileset_path, tile, ext):
             pretty=True,
             tile_opener=tile_opener)
 
-    def max_proj(self, *dims):
+    def max_proj(self, *dims: Indices) -> numpy.ndarray:
+        """return a max projection over one or more axis of the image tensor
+
+        Parameters
+        ----------
+        dims : Tuple[Indices]
```

Comment:
is this right?  Is it considered a tuple?

Reply:
Mypy says it should just be documented as Indices, so I will change it to that. However, yes, it will be treated as a tuple by python.